cmd/kubectl-datadog/clusteragent/upgrade/upgrade_test.go

```diff
@@ -5,154 +5,156 @@
 
 package upgrade
 
-import (
-	"context"
-	"fmt"
-	"testing"
+// TODO add test for v2alpha1
```

Comment:
I intentionally left this file with the v1alpha1 tests commented out to make it easier to add the v2alpha1 tests in the near future